client/src/Validation/validation.js

```
@@ -483,8 +483,71 @@
+			then: joi.string().required().messages({
+				"string.empty": "Password cannot be empty when using username/password authentication",
+				"any.required": "Password is required for username/password authentication",
+			}),
+			otherwise: joi.string().optional().allow(""),
+		}),
+		otherwise: joi.forbidden(),
+	}),
+
+	ntfyBearerToken: joi.when("type", {
+		is: "ntfy",
+		then: joi.when("ntfyAuthMethod", {
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)